pkg/rebuild/schema/schema.go

```diff
@@ -222,18 +222,15 @@ type CreateRunResponse struct {
 
 // SmoketestAttempt stores rebuild and execution metadata on a single smoketest run.
 type SmoketestAttempt struct {
-	Ecosystem         string  `firestore:"ecosystem,omitempty"`
-	Package           string  `firestore:"package,omitempty"`
-	Version           string  `firestore:"version,omitempty"`
-	Artifact          string  `firestore:"artifact,omitempty"`
-	Success           bool    `firestore:"success,omitempty"`
-	Message           string  `firestore:"message,omitempty"`
-	Strategy          string  `firestore:"strategy,omitempty"`
-	TimeCloneEstimate float64 `firestore:"time_clone_estimate,omitempty"`
-	TimeSource        float64 `firestore:"time_source,omitempty"`
-	TimeInfer         float64 `firestore:"time_infer,omitempty"`
-	TimeBuild         float64 `firestore:"time_build,omitempty"`
-	ExecutorVersion   string  `firestore:"executor_version,omitempty"`
-	RunID             string  `firestore:"run_id,omitempty"`
-	Created           int64   `firestore:"created,omitempty"`
+	Ecosystem       string          `firestore:"ecosystem,omitempty"`
+	Package         string          `firestore:"package,omitempty"`
+	Version         string          `firestore:"version,omitempty"`
+	Artifact        string          `firestore:"artifact,omitempty"`
+	Success         bool            `firestore:"success,omitempty"`
+	Message         string          `firestore:"message,omitempty"`
+	Strategy        StrategyOneOf   `firestore:"strategy,omitempty"`
+	Timings         rebuild.Timings `firestore:"timings,omitempty"`
```

Comment:
Will this break backwards compatibility? IIUC currently we store things in TimeSource TimeInfer and TimeBuild fields, but I think this change might push all of those into a single Timings field. And maybe this will fail to read timings from old firestore entries?

It's probably fine to break backwards compatibility, just wondering if you have thoughts.

Reply:
Yeah this will break the existing fields. we could add the existing ones back to support the current data but I wasn't sure how important that was. i can hack it together and see how it looks.

Reply:
Discussed offline, I think the latest version will ignore old formats gracefully, leaving the new fields empty (timings and strategy).